Add documentation for both users and developers in README.md

Cool 👍 

I installed Nodejs and npm without a problem, but the browserify command throws this:

`C:\Users\xxxxx>browserify background-pre.js -o background.js
Error: Can't walk dependency graph: Cannot find module 'C:\Users\xxxxx\background-pre.js' from 'C:\Users\xxxxx\_fake.js'
    required by C:\Users\xxxxx\_fake.js
    at C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:136:35
    at load (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:155:43)
    at onex (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:180:17)
    at C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:15:69
    at FSReqCallback.oncomplete (fs.js:168:21)`

I did now! Silly me. But it's still having an issue finding stuff:

`C:\Users\xxxxx\AppData\Roaming\Mozilla\Firefox\Profiles\dev-edition-default\extensions\ff-pixel-helper-master>browserify background-pre.js -o background.js
Error: Can't walk dependency graph: Cannot find module 'query-string' from 'C:\Users\xxxxx\AppData\Roaming\Mozilla\Firefox\Profiles\dev-edition-default\extensions\ff-pixel-helper-master\background-pre.js'
    required by C:\Users\xxxxx\AppData\Roaming\Mozilla\Firefox\Profiles\dev-edition-default\extensions\ff-pixel-helper-master\background-pre.js
    at C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:115:35
    at processDirs (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:268:39)
    at isdir (C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:275:32)
    at C:\Users\xxxxx\AppData\Roaming\npm\node_modules\browserify\node_modules\resolve\lib\async.js:25:69
    at FSReqCallback.oncomplete (fs.js:168:21)`

--- a/README.md
+++ b/README.md
@@ -1,16 +1,41 @@
-# Firefox Facebook Pixel Helper
+# FF Pixel Helper
+
+Unofficial Facebook Pixel Helper extension for Firefox. View all Facebook events fired on the current page.
 
 ## Introduction
 
-WIP
+This Firefox extension emulates the [original Facebook Pixel Helper extension](https://chrome.google.com/webstore/detail/facebook-pixel-helper/fdgfkebogiimcoedlicjlajpkdmockpc) made by Facebook, which is only available in Chrome.
+
+Please note that this extension is fairly experimental and can't guarantee the same accuracy level as the original one. In any case, if you encounter any bug or have any suggestion, feel free to report it on the [Issues tab](https://github.com/angelbt91/ff-pixel-helper/issues). 
+
+This extension is not endorsed nor recommended by Facebook.
 
 ## Deployment
 
-`npm install`
+Follow these steps if you want to deploy a local copy of the extension from its Github repository.
 
-`npm install -g browserify`
+Clone this repository and get into the folder:
+```
+$ git clone https://github.com/angelbt91/ff-pixel-helper.git
+$ cd ff-pixel-helper
+```
 
-`browserify background-pre.js -o background.js`
 
-`cd popup && npm install & npm run build`
+Install dependencies:
 
+`$ npm install`
+
+Install the Browserify util globally, in order to generate the background script:
+```
+$ npm install -g browserify
+$ browserify background-pre.js -o background.js
+```
+
+Get into the popup folder, install dependencies and generate the static files:
+``` 
+$ cd popup
+$ npm install
+$ npm run build
+```
+
+This local copy of the extension can be tested by loading it on Firefox as a [temporary addon](https://extensionworkshop.com/documentation/develop/temporary-installation-in-firefox/).

--- a/manifest.json
+++ b/manifest.json
@@ -1,8 +1,8 @@
 {
 
   "manifest_version": 2,
-  "name": "Firefox Facebook Pixel Helper",
-  "version": "1.0.1",
+  "name": "FF Pixel Helper",
+  "version": "1.0.2",
 
   "description": "Unofficial Facebook Pixel Helper extension for Firefox. View all Facebook events fired on the current page.",
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "ff-pixel-helper",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Unofficial Facebook Pixel Helper extension for Firefox. View all Facebook events fired on the current page.",
   "main": "background.js",
   "scripts": {

